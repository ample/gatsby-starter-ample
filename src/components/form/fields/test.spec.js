import React from "react"
import renderer from "react-test-renderer"

import { fieldMap, FormFields } from "./"

describe("FormFields", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<FormFields fields={[]} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe("fieldMap", () => {
  it("pulls keys from config", () => {
    expect(Object.keys(fieldMap).sort()).toEqual(["select", "text"])
  })
})
