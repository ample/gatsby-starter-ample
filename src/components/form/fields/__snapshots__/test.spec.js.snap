// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormFields renders correctly 1`] = `
<div
  className="form_field_group"
/>
`;
